.travis.yml

```diff
@@ -9,24 +9,26 @@ addons:
     - ccache
     - cmake
     - swig
+    - libxkbcommon-x11-0
     - libglu1-mesa-dev
-
+    - libxcb-icccm4
+    - libxcb-image0
+    - libxcb-keysyms1
+    - libxcb-randr0
+    - libxcb-render-util0
+    - libxcb-xinerama0
+    
 env:
   global:
     - INSTALL_EDM_VERSION=2.0.0
       PYTHONUNBUFFERED="1"
+      QT_DEBUG_PLUGINS="1"
 
 matrix:
   include:
-  - env: RUNTIME=2.7 TOOLKITS="null pyqt pyside wx"
-  - os : osx
-    env: RUNTIME=2.7 TOOLKITS="null pyqt pyside wx"
-  - env: RUNTIME=3.5 TOOLKITS="null pyqt"
-  - os : osx
-    env: RUNTIME=3.5 TOOLKITS="null pyqt"
-  - env: RUNTIME=3.6 TOOLKITS="null pyqt"
+  - env: RUNTIME=3.6 TOOLKITS="null pyqt pyqt5"
```

Comment:
do we want to test pyside2 on osx/linux like we are on win?

Reply:
yes!